### ISSUE TITLE
一点赞,就会报错

### ISSUE THREAD

**Issue by qxl1231** [soliury/noder-react-native#47]
报网络原因的错误
![wechat_1464859735](https://cloud.githubusercontent.com/assets/8305742/15740461/7f7981d8-28e7-11e6-852d-513374933967.jpeg)

**Comment by TakWolf** [comment]
给自己点赞或者accessToken过期，状态码都会返回403

**Comment by TakWolf** [comment]
@soliury 现在所有接口的error_msg返回的都是中文，可以直接拿来做提示

**Comment by qxl1231** [comment]
确实,验证下来给别人点赞ok的,就是给自己点赞不行,那不如直接不要在自己说的话下面加点赞按钮咯,还有返回的错误码提示也不对,这不是网络错误啊

**Comment by TakWolf** [comment]
@qxl1231 点赞按钮旁边有该回复的点赞数量，不能去掉

**Comment by qxl1231** [comment]
那就做个条件判断咯,如果这里的user=本人,则把这个按钮置灰,不可点击,至少不能让用户报错撒

**Comment by soliury** [comment]
我记得第一版是有判断的，后来重构的时候貌似就漏掉了，这个放到下一版更新吧
